content/01.abstract.md

```diff
@@ -1,3 +1 @@
 ## Abstract {.page_break_before}
-
-
```

Comment:
This causes conflict. I see your text editor is pathological. Perhaps we should add comments to the end of files to stop this pathological behavior.

Reply:
#4 resolves this issue